src/main/java/calculator/Numbers.java

```diff
@@ -0,0 +1,44 @@
+package calculator;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.stream.Collectors;
+
+public class Numbers {
+
+    private List<Integer> numbers = new ArrayList<>();
+
+    public Numbers() {
+    }
+
+    public Numbers(List<String> numbers) {
+        this.numbers = numbers.stream()
+                .map(Numbers::convertNumber)
+                .collect(Collectors.toList());
```

Comment:
자바 16 이후부터는 이렇게 변경할 수 있을것 같아요.
```suggestion
.toList();
```